webapps/frontend/app/widgets/lazylist/view.html

```diff
@@ -28,31 +28,38 @@
         <!-- /ko -->
     <!-- /ko -->
     <!-- ko if: displaymode() === 'table' -->
-        <table class="table table-striped table-hover" >
-            <thead data-part="head"></thead>
-            <!-- ko if: items().length === 0 -->
-                <tbody>
-                    <tr>
-                        <td data-bind="attr: { colspan: colspan() }">
-                            <i data-i18n="ovs:generic.noentries"></i>
-                        </td>
-                    </tr>
-                </tbody>
-            <!-- /ko -->
-            <!-- ko if: items().length > 0 -->
-                <tbody data-bind="foreach: items">
-                    <!-- ko if: !$parent.isLoaded($data) -->
+        <!-- ko if: !itemsHaveBeenLoaded($parent) -->
+            <div>
```

Comment:
Might be better to keep it as table (with e.g. the headers displayed) just like all our other tables.